services/web/server/src/simcore_service_webserver/projects/_tags_api.py

```diff
@@ -32,6 +34,20 @@ async def add_tag(
     project: ProjectDict = await db.add_tag(
         project_uuid=f"{project_uuid}", user_id=user_id, tag_id=int(tag_id)
     )
+
+    if project["workspaceId"] is not None:
+        workspace_db: UserWorkspaceAccessRightsDB = (
+            await workspaces_db.get_workspace_for_user(
+                app=app,
+                user_id=user_id,
+                workspace_id=project["workspaceId"],
+                product_name=product_name,
+            )
+        )
+        project["accessRights"] = {
+            key: access.dict() for key, access in workspace_db.access_rights.items()
```

Comment:
same here

Reply:
yes, I might do it in upcoming PR, as there are more places where are use the same thing.